Code/Tools/ProjectManager/Source/Application.h

```diff
@@ -13,6 +13,7 @@
 #include <PythonBindings.h>
 #include <Settings.h>
 #include <ProjectManagerWindow.h>
+#include <DownloadController.h>
```

Comment:
do you need this? it is included in the .cpp file
```suggestion
```

Reply:
It's not needed in the cpp file here either since I made the creation of the object elsewhere. Removed both includes.